alvr/common/src/data/settings.rs

```diff
@@ -270,6 +270,8 @@ pub struct HeadsetDesc {
 pub enum SocketProtocol {
     Udp,
     Tcp,
+    #[schema(advanced, min = 1.0, step = 0.1, gui = "UpDown")]
+    ThrottledUdp(f32),
```

Comment:
It is an advanced setting meant to not be changed, but I agree that like this it's very confusing. I separated the setting so it can be given a name and a description.